.github/actions/build-cached-packs/action.yaml

```diff
@@ -0,0 +1,43 @@
+name: "Build with cached packs"
+inputs:
+  gh-token:
+    description: "GitHub Token for authentication"
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install jq (JSON processor)
+      run: |
+        sudo apt-get update && sudo apt-get install -y jq
```

Comment:
Are we installing jq becuase of the self-hosted runner? By default, it comes with the action runners. The other point is doing an apt-get update && sudo apt-get install every time will slow things down a bit and is unnecessary in my opinion.

Reply:
Good point about the update I will remove it.. I need jq to look up the workflow id in the next action. 

```
 run_id=$(gh run list --workflow="post_release.yaml" --limit 1 --json databaseId | jq -r '.[0].databaseId')
```

Reply:
I checked the self-hosted runner. We already have jq installed there and have been using it for a while with the screenshot_artifact.sh

Reply:
This used to fail on the pull request build. I don't think they run on the self-hosted runner do they?